docs/date.md

```diff
@@ -32,8 +32,7 @@ All values are given as reckoned in the [ISO 8601 calendar](https://en.wikipedia
 Together, `isoYear`, `isoMonth`, and `isoDay` must represent a valid date in that calendar.
 
 The range of allowed values for this type is exactly enough that calling [`getDate()`](./datetime.html#getDate) on any valid `Temporal.DateTime` will succeed.
-If `isoYear`, `isoMonth`, and `isoDay` form a date outside of this range, then `constrain` mode will clamp the date to the limit of the allowed range.
-Both `balance` and `reject` mode will throw a `RangeError` in this case.
+If `isoYear`, `isoMonth`, and `isoDay` form a date outside of this range, a `RangeError` will be thrown.
```

Comment:
The text about `constrain` is still relevant, I believe.

Reply:
Good catch. Fixed this by stealing similar content from the `DateTime` constructor docs.  Resolving.